docs/platform/calibration_guide.md

```diff
@@ -0,0 +1,154 @@
+# Field Calibration Guide
+
+
+# Contents
```

Comment:
@HassanBahati Could you add links to the items in the content (table of content) section?

Reply:
@Codebmk i have added the links to the items in the table of content